Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -373,7 +373,7 @@ private class ClearPrivateDataSetting: Setting {
         self.profile = settings.profile
         self.tabManager = settings.tabManager
 
-        let clearTitle = NSLocalizedString("Clear private data", comment: "Clear private data setting")
+        let clearTitle = NSLocalizedString("Clear private data", comment: "Title shown on the clear private data setting. Clicking it will open a dialog prompting the user to make sure they want to clear all of their private data.")
```

Comment:
Alternative comment: ```Label used as an item in Settings. When touched it will open a dialog prompting the user to make sure they want to clear all of their private data.```.